stage1/init/common/pod.go

```diff
@@ -348,6 +380,18 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		opts = append(opts, unit.NewUnitOption("Service", "SyslogIdentifier", filepath.Base(app.Exec[0])))
 	}
 
+	if flavor == "kvm" {
+		capabilities := kvm.GetAppCapabilities(app)
+		capabilities = append(capabilities, stdCapabilities...)
```

Comment:
Well this configuration file is great, but AFAIK each app inside pod could has got own capabilities configuration, so setting one common file for every app won't work.